packages/menu/src/Menu/Menu.js

```diff
@@ -16,18 +16,22 @@ function Menu({
   buttonNode,
   buttonProps,
   buttonText,
+  contentProps,
   color,
   width,
   height,
   size,
   trapFocus,
+  placement,
   children,
   ...props
 }) {
   const MenuContent = ({ handleClose }) => (
-    <MenuList id={id} color={color} size={size} height={height} handleClose={handleClose}>
-      {children}
-    </MenuList>
+    <Box {...contentProps}>
```

Comment:
can we not pass `contentProps` directly to the MenuList?

Reply:
I don't pass it to `MenuList` since it has it's own styling https://github.com/priceline/design-system/blob/f53e03e2a7247666a1b9d1ca303c62cc3df779da/packages/menu/src/MenuList/MenuList.js#L71-L83

And overriding those styling would mess with the placement of the children. Especially when adjusting margin and padding.

Reply:
Discussed this offline. We don't need adjustment at all so removing `contentProps`